cms/settings/base.py

```diff
@@ -276,7 +276,7 @@
 }
 
 # Place static files that need a specific URL (such as robots.txt and favicon.ico) in the "public" folder
-WHITENOISE_ROOT = BASE_DIR / "public"
+# WHITENOISE_ROOT = BASE_DIR / "public"
```

Comment:
This is commented out because we're not using the functionality. When we need it, we can easily add it back. Commenting it out removes a warning when running tests.

Reply:
let's remove it all together. We can re-add, if and when we need it